packages/eslint-plugin-fiori-tools/README.md

```diff
@@ -0,0 +1,22 @@
+# @sap-ux/eslint-plugin-fiori-tools
+
+Custom linting plugin for Fiori tools apps
```

Comment:
Wouldn't it be for all UI5 apps?

Reply:
Updated in https://github.com/SAP/open-ux-tools/pull/984/commits/0d0de75fcf865cc43d3d90a34d4975b27254c20a